### PR TITLE
Update local ollama doc

### PR DESCRIPTION
**Pull request by enyst**
**CHANGELOG**

Update local Ollama docs. Since the app has changed, some settings have moved from files to the UI, and other tweaks.

---
Cc: @mamoodi

### CODE REVIEW

File: docs/modules/usage/llms/local-llms.md [hunk 1]
```diff
@@ -66,6 +63,15 @@ docker run \
 ```
 
 You should now be able to connect to `http://localhost:3000/`
+### Configure the Web Application
+
+When running `openhands`, you'll need to set the following in the OpenHands UI through the Settings:
+- the model to `ollama/&lt;model-name&gt;`
```
Comment (mamoodi) [inline docs/modules/usage/llms/local-llms.md:69]: You will have to remove the `` otherwise the <> doesn't get rendered

**Comment by mamoodi** [comment]
Does this work if you follow the doc exactly as shown? Did you test it? Cause that would be great if we can refer to people to this doc just in case they want to test it.

File: docs/modules/usage/llms/local-llms.md [hunk 2]
```diff
@@ -66,6 +63,15 @@ docker run \
 ```
 
 You should now be able to connect to `http://localhost:3000/`
+### Configure the Web Application
```
Comment (tobitege) [inline docs/modules/usage/llms/local-llms.md:66]: Add empty line before line 66

**Review by mamoodi — APPROVED** [review]

**Comment by enyst** [comment]
> Does this work if you follow the doc exactly as shown? Did you test it? Cause that would be great if we can refer to people to this doc just in case they want to test it.

Of course. 🧸 

Sorry, maybe it's not clear what's supposed to be happening here, let me put this differently:
- as with other docs, we meet here the fact that we added API key and base URL to the UI. That implies that they're NOT needed anymore: 1) for running with docker run, with -e lines, 2) for running with make run, in config.toml. They won't hurt if they remain in these places too, but it will be confusing to people that they set them and they're now _ignored_. We can guide folks to set them in the UI settings instead.
- specific to this doc, is that with `docker run` base url needs to be http://host.docker.internal:11434, while when running in dev mode with `make run` it needs to be `http://localhost:11434'.

**Pull request merged**